.vitepress/config.ts

```diff
@@ -126,6 +126,13 @@ export default defineConfig({
       placement: 'vitejsdev',
     },
 
+    outlineTitle: '目次',
+
+    docFooter: {
+      prev: '前へ',
+      next: '次へ',
```

Comment:
こちら Vue.js の日本語ドキュメントと同じ文言に変更お願いします:pray:
```suggestion
      prev: '前のページ',
      next: '次のページ',
```

Reply:
サジェスチョンありがとうございます！
確かにVue.jsの日本語ドキュメントだとそう書いてますね。
次回からそっちも確認してからプルリク送ります👍